src/collective/nitf/tests/test_content.robot

```diff
@@ -73,9 +79,6 @@ Create News Article, subobjects and test views
     Go To  ${PLONE_URL}/miracle-cure/@@galleria
     Page Should Contain  Miracle Cure
 
-    Go To  ${PLONE_URL}/miracle-cure/@@nitf
-    Page Should Contain  Miracle Cure
```

Comment:
This test is odd, the text was into the view, but Robot couldn't find it.  Maybe that's because the view returns a XML document, not HTML